torch/linalg/__init__.py

```diff
@@ -483,6 +483,79 @@
     >>> tensor([ 11.6734+0.j, 105.1037+0.j,  10.1978+0.j])
 """)
 
+pinv = _add_docstr(_linalg.linalg_pinv, r"""
+linalg.pinv(input, rcond=1e-15, hermitian=False) -> Tensor
```

Comment:
Note for @heitorschueroff, we should be sure to docs deprecate the old pinverse before 1.8.

Reply:
The NumPy version of this operator can throw a Runtime error:

https://numpy.org/doc/1.18/reference/generated/numpy.linalg.pinv.html?highlight=pinv#numpy.linalg.pinv

The other linear algebra functions that we've recently added also explain when they may throw runtime errors. Is there something we can document here, too?

Reply:
I added a note that if svd or eigh do not converge then the runtime error will be thrown. It's difficult to test these situations though.

Reply:
What makes that hard to test?

Reply:
I can't find how to generate matrices that would make svd or eigh fail. They're quite robust.